### ISSUE TITLE
Missing focus state of upload button in SASS themes

### ISSUE THREAD

**Issue by magdalenaan** [telerik/kendo-themes#3668]
**To reproduce**

1. Go to https://demos.telerik.com/kendo-ui/upload/basicusage
1. Try to focus the upload button with keyboard navigation
1. Focus state is not visible in SASS themes

**Expected behavior**

**Affected package**
* All

**Affected suites**
* Kendo UI for jQuery

**Affected browsers**
* All

**Comment by TeyaVes** [comment]
@magdalenaan , When using keyboard navigation the class added to the button in Kendo jQuery is _.k-state-focused_. The correct class that should be added is **.k-focus**, which would add the correct styling.

related to: https://github.com/telerik/kendo-themes/issues/3595
